src/app/header/header.component.ts

```diff
@@ -116,8 +116,8 @@ export class HeaderComponent implements OnInit, OnDestroy {
   }
 
   private getFullName(user: User): string {
-    return `${user.lastName} ${user.firstName.slice(0, 1)}.${
-      user.middleName ? user.middleName.slice(0, 1) + '.' : ''
+    return `${user?.lastName} ${user?.firstName.slice(0, 1)}.${
```

Comment:
It would be better to made one check instead four.

Reply:
Removed this changes completely